Assets/Scripts/Models/Functions/FunctionsManager.cs

```diff
@@ -112,6 +112,9 @@ public static void JobComplete_FurnitureBuilding(Job theJob)
     {
         WorldController.Instance.World.PlaceFurniture(theJob.JobObjectType, theJob.tile);
 
+        // Let our workspot tile know it is no longer reserved for us
+        WorldController.Instance.World.UnreserveTileAsWorkSpot((Furniture)theJob.buildablePrototype, theJob.tile);
```

Comment:
I don't get why you unreserve it here. Shouldn't it still be reserved so that other furniture can't be placed in this spot unless is a special kind of furniture like a chair?